	FROM cte
	WINDOW w AS (ORDER BY x rows between 1 preceding and 1 following);
2022-08-01 21:57:31.340 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 106
2022-08-01 21:57:31.340 UTC [28304] STATEMENT:  WITH cte (x) AS (
	        select 1 union all select 1 union all select 1 union all
	        SELECT * FROM generate_series(5, 49, 2)
	)
	SELECT x, (sum(x) over w)
	FROM cte
	WINDOW w AS (ORDER BY x range between 1 preceding and 1 following);
2022-08-01 21:57:31.340 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.340 UTC [28304] STATEMENT:  CREATE FUNCTION t2(val anyelement, n integer = 1) RETURNS anyelement
	  LANGUAGE internal WINDOW IMMUTABLE STRICT AS 'window_nth_value';
2022-08-01 21:57:31.341 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.341 UTC [28304] STATEMENT:  CREATE FUNCTION auth_tbl(text, anyelement) RETURNS text AS
	$$ SELECT COALESCE($1, '') || '+' || quote_nullable($2) $$
	LANGUAGE SQL IMMUTABLE;
2022-08-01 21:57:31.341 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.341 UTC [28304] STATEMENT:  CREATE AGGREGATE uname (anyelement)
	(
		stype = text,
		sfunc = auth_tbl,
		mstype = text,
		msfunc = auth_tbl,
		minvfunc = auth_tbl
	);
2022-08-01 21:57:31.341 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.341 UTC [28304] STATEMENT:  CREATE FUNCTION credential(text, anyelement) RETURNS text AS
	$$ SELECT $1 || '+' || quote_nullable($2) $$
	LANGUAGE SQL STRICT IMMUTABLE;
2022-08-01 21:57:31.341 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.341 UTC [28304] STATEMENT:  CREATE FUNCTION label(int4, int4) RETURNS int4 AS
	$$ SELECT CASE WHEN random() < 0.2 THEN NULL ELSE $1 - $2 END $$
	LANGUAGE SQL STRICT;
2022-08-01 21:57:31.341 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.341 UTC [28304] STATEMENT:  CREATE AGGREGATE foo_tbl (int4)
	(
		stype = int4,
		sfunc = int4pl,
		mstype = int4,
		msfunc = int4pl,
		minvfunc = label
	);
2022-08-01 21:57:31.341 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 59
2022-08-01 21:57:31.341 UTC [28304] STATEMENT:  WITH
	vs AS (
		SELECT i, (random() * 100)::int4 AS v
		FROM generate_series(1, 100) AS i
	),
	sum_following AS (
		SELECT i, SUM(v) OVER
			(ORDER BY i DESC ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW) AS s
		FROM vs
	)
	SELECT DISTINCT
		sum_following.s = foo_tbl(v) OVER fwd AS eq1,
		-sum_following.s = foo_tbl(-v) OVER fwd AS eq2,
		100*3+(vs.i-1)*3 = length(uname(''::text) OVER fwd) AS eq3
	FROM vs
	JOIN sum_following ON sum_following.i = vs.i
	WINDOW fwd AS (
		ORDER BY vs.i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING
	);
2022-08-01 21:57:31.341 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 17
2022-08-01 21:57:31.341 UTC [28304] STATEMENT:  SELECT i,AVG(v::numeric) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(1,1.5),(2,2.5),(3,NULL),(4,NULL)) t(i,v);
2022-08-01 21:57:31.341 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.341 UTC [28304] STATEMENT:  SELECT i,SUM(v::money) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(1,'1.10'),(2,'2.20'),(3,NULL),(4,NULL)) t(i,v);
2022-08-01 21:57:31.341 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 20
2022-08-01 21:57:31.341 UTC [28304] STATEMENT:  SELECT VARIANCE(n::bigint) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(1,600),(2,470),(3,170),(4,430),(5,300)) r(i,n);
2022-08-01 21:57:31.342 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 20
2022-08-01 21:57:31.342 UTC [28304] STATEMENT:  SELECT VARIANCE(n::int) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(1,600),(2,470),(3,170),(4,430),(5,300)) r(i,n);
2022-08-01 21:57:31.342 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 20
2022-08-01 21:57:31.342 UTC [28304] STATEMENT:  SELECT VARIANCE(n::numeric) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(1,600),(2,470),(3,170),(4,430),(5,300)) r(i,n);
2022-08-01 21:57:31.342 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 17
2022-08-01 21:57:31.342 UTC [28304] STATEMENT:  SELECT i,SUM(v::int) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND CURRENT ROW)
	  FROM (VALUES(1,1),(2,2),(3,NULL),(4,NULL)) t(i,v);
2022-08-01 21:57:31.342 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 17
2022-08-01 21:57:31.342 UTC [28304] STATEMENT:  SELECT i,SUM(v::int) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND 1 FOLLOWING)
	  FROM (VALUES(1,1),(2,2),(3,NULL),(4,NULL)) t(i,v);
2022-08-01 21:57:31.342 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 104
2022-08-01 21:57:31.342 UTC [28304] STATEMENT:  SELECT a, b,
	       SUM(b) OVER(ORDER BY A ROWS BETWEEN 1 PRECEDING AND CURRENT ROW)
	FROM (VALUES(1,1::numeric),(2,2),(3,'NaN'),(4,3),(5,4)) t(a,b);
2022-08-01 21:57:31.342 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.342 UTC [28304] STATEMENT:  RESET SESSION AUTHORIZATION;
2022-08-01 21:57:31.342 UTC [28304] WARNING:  there is no transaction in progress
2022-08-01 21:57:31.342 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.342 UTC [28304] STATEMENT:  DISCARD ALL;
2022-08-01 21:57:31.342 UTC [28304] ERROR:  could not find pg_class tuple for index 2662 at character 24
2022-08-01 21:57:31.342 UTC [28304] STATEMENT:  SELECT n.nspname 
	FROM pg_catalog.pg_namespace n 
	WHERE n.nspname !~ '^pg_' AND n.nspname <> 'information_schema';
2022-08-01 21:57:31.342 UTC [28304] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 21:57:31.342 UTC [28304] STATEMENT:  CREATE SCHEMA public; SET search_path TO public;
2022-08-01 21:57:32.736 UTC [30548] PANIC:  could not open critical system index 2662
2022-08-01 21:57:32.738 UTC [27976] LOG:  server process (PID 30548) was terminated by signal 6: Aborted
2022-08-01 21:57:32.738 UTC [27976] LOG:  terminating any other active server processes
2022-08-01 21:57:32.762 UTC [27976] LOG:  all server processes terminated; reinitializing
2022-08-01 21:57:32.773 UTC [30549] LOG:  database system was interrupted; last known up at 2022-08-01 21:57:12 UTC
2022-08-01 21:57:32.788 UTC [30549] LOG:  database system was not properly shut down; automatic recovery in progress
2022-08-01 21:57:32.788 UTC [30549] LOG:  redo starts at 0/16B53C0
2022-08-01 21:57:32.855 UTC [30549] LOG:  invalid record length at 0/19BCF00: wanted 24, got 0
2022-08-01 21:57:32.855 UTC [30549] LOG:  redo done at 0/19BCEC8 system usage: CPU: user: 0.05 s, system: 0.00 s, elapsed: 0.06 s
2022-08-01 21:57:32.883 UTC [27976] LOG:  database system is ready to accept connections
2022-08-01 21:57:33.597 UTC [30566] WARNING:  there is no transaction in progress
2022-08-01 21:57:33.656 UTC [30572] ERROR:  table "hash_split_heap" does not exist
